src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -688,6 +706,15 @@ internal Collection<FunctionDescriptor> GetFunctionDescriptors(IEnumerable<Funct
             return functionDescriptors;
         }
 
+        internal static bool IsSingleLanguage(IEnumerable<FunctionMetadata> functions)
```

Comment:
This can probably move elsewhere. We already have a few classes using this. `Utility` perhaps?